### PR TITLE
Assignment_Q4_Kunal

### PR DESCRIPTION
**Pull request by KunalKTechnocredits**
Q4. WAP to find the area and perimeter of Rectangle & Circle

### CODE REVIEW

File: JavaAssignment/src/kunal/PerimeterArea.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package kunal;
+public class PerimeterArea 
+{
+    int radius, length, breadth;
+    double pi = 3.14;
+	double perimeter;
+	double Area;
+	double Area1;
+   
+    void circle(int radius)
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/PerimeterArea.java:10]: Give proper method name, not getting what are you going to do with "circle" method. If you are finding an area of circle write like "areaOfCircle" as a method name.

File: JavaAssignment/src/kunal/PerimeterArea.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package kunal;
+public class PerimeterArea 
+{
+    int radius, length, breadth;
+    double pi = 3.14;
+	double perimeter;
+	double Area;
+	double Area1;
+
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/PerimeterArea.java:9]: Add a comment in here like what are you going to achieve with this method.

File: JavaAssignment/src/kunal/PerimeterArea.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+   
+    void circle(int radius)
+    {
+        perimeter = 2 * pi * radius;
+		Area1= pi * radius * radius;
+        System.out.println("Perimeter of circle:"+perimeter);
+		System.out.println("Area of circle:"+Area1);
+    } 
+    
+    void rectangle(int length,int breadth)
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/PerimeterArea.java:18]: if you are finding an area of rectangle write like "areaOfRectangle" as a method name. And comment before this method.

File: JavaAssignment/src/kunal/PerimeterArea.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        perimeter = 2 * (length + breadth);
+		Area= length * breadth;
+        System.out.println("Perimeter of rectangle:"+perimeter);
+		System.out.println("Area of rectangle:"+Area);
+    }
+   
+    public static void main(String[] args)
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/PerimeterArea.java:26]: Format your code before committing .

**Review by mkanani13 — APPROVED** [review]
i think you didn't work on the comments
still approving your PR.

**Pull request merged**